linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -9,7 +9,8 @@
    "source": [
```

Comment:
<p>Would perhaps the Alternative Name NumIdent be the analogous nickname logic for this part of the process?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>No. The Alternate Name Numident, as well as the Name/DOB reference file, includes <em>alternate</em> names -- that is, names that actually appeared in some record for this person. <em>Nickname </em>processing on the other hand applies known rules about how names relate -- e.g. we know that someone named Will is (probably) named William, even if they <em>never</em> use William.</p>